AEPMessaging/Sources/LiveActivity/LiveActivityUpdateTokenStore.swift

```diff
@@ -0,0 +1,84 @@
+/*
+ Copyright 2025 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPServices
+
+final class LiveActivityUpdateTokenStore: PersistedMapStoreBase<LiveActivity.UpdateTokenMap> {
+    init() {
+        super.init(storeKey: MessagingConstants.NamedCollectionKeys.LIVE_ACTIVITY_UPDATE_TOKENS)
```

Comment:
We can write a TODO here for future PR: 

During initializing we might want to discard the expired tokens. (Live Activity update tokens can expire after every 12 hours)